### PR TITLE
Wazo 2840 remove python2 build

### PR DESCRIPTION
**Pull request by sopelj**
Depends-on: https://github.com/wazo-platform/wazo-lib-rest-client/pull/12

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -5,27 +5,11 @@ Maintainer: Wazo Maintainers <dev@wazo.community>
 Build-Depends:
  debhelper (>= 9),
  dh-python,
- python-all (>= 2.7),
- python-setuptools,
  python3-all,
  python3-setuptools
 Standards-Version: 3.9.6
-X-Python-Version: >= 2.7
 X-Python3-Version: >= 3.4
```
Comment (pc-m) [inline debian/control:11]: Let's make it python 3.7 since we are using features that were not available in python 3.4 (f-strings)

**Review by pc-m — CHANGES_REQUESTED** [review]
Good job Jessie, I LOVE seeing python 2 being removed. :heart_eyes_cat:

File: debian/control (hunk 1 above)
Comment (sopelj) [reply debian/control:11]: Oh yes, good point! I thought I had changed all of them, but I missed this one. Thanks!

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
regate

**Pull request merged**